<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGroupDialog</class>
 <widget class="QDialog" name="AddGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QPushButton" name="verifyButton">
     <property name="text">
      <string>Verify</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0">
     <item row="3" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="5">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="3">
         <widget class="QComboBox" name="fideRtgComboBox"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="famNameLabel">
          <property name="text">
           <string>Family Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="genderComboBox"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="natRtgLabel">
          <property name="text">
           <string>National Rating</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="natRtgComboBox"/>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="fideRtgLabel">
          <property name="text">
           <string>FIDE Rating</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="birthDateComboBox"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="nameComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="natIdLabel">
          <property name="text">
           <string>National ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="birthDateLabel">
          <property name="text">
           <string>Birthdate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="genderLabel">
          <property name="text">
           <string>Gender</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="natIdComboBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="fideIdLabel">
          <property name="text">
           <string>FIDE ID</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="fideIdComboBox"/>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="famNameComboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="teamsComboBox"/>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="sectionLabel">
          <property name="text">
           <string>Section</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QComboBox" name="sectionComboBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="teamsLabel">
          <property name="text">
           <string>Teams</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3" colspan="5">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0,0" columnstretch="0,0,0,0,0">
        <item row="1" column="0">
         <widget class="QLabel" name="filePathLabel">
          <property name="text">
           <string>File Path</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="1">
            <widget class="QComboBox" name="fieldSepComboBox"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="fieldSepLabel">
             <property name="text">
              <string>Field Separator</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="filePathEdit"/>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="headersCheckBox">
          <property name="text">
           <string>Headers</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QPushButton" name="autoButton">
          <property name="text">
           <string>Automate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
